jsrc/util/stringx.h

```diff
@@ -0,0 +1,8 @@
+
+#pragma once
+
+#include <string>
+
+namespace util{
+    auto remove_all_occurrences(std::string &str, std::string_view const &sub_str) -> void;
```

Comment:
`std::string_view` doesn't need to be `const&` because it is non-owning :slightly_smiling_face: